pipelinewise/fastsync/commons/split_gzip.py

```diff
@@ -0,0 +1,127 @@
+"""Functions that write chunked gzipped files."""
+import io
+import logging
+import gzip
+
+LOGGER = logging.getLogger(__name__)
+
+DEFAULT_CHUNK_SIZE_MB = 1000
```

Comment:
Can we use these constants in the `pipelinewise/cli/config.py`? the defaults are current defined in two places which would make it hard to update the default later

Reply:
Same as for DEFAULT_MAX_CHUNKS

Reply:
we should always update only `pipelinewise/cli/config.py`.

The values in this file are not PPW specific, but defaults for the SplitGzipFile file-like object as an independent object. Basically this could be hosted completely outside of PPW as an external dependency.

Reply:
hmm ok